Sloth.Web/Controllers/TransactionsController.cs

```diff
@@ -233,6 +237,43 @@ public async Task<IActionResult> CreateReversal(string id)
             return RedirectToAction("Details", new { id = reversal.Id });
         }
 
+        [HttpPost]
+        public async Task<IActionResult> CallWebHook(string id)
+        {
+            var transaction = await DbContext.Transactions
+                .Include(t => t.Source)
+                .ThenInclude(s => s.Team)
+                .FirstOrDefaultAsync(t => t.Id == id);
```

Comment:
nitpicky but I like `SingleOrDefaultAsync()` when looking for IDs.    juuuuuust in case.

Reply:
Done